src/Sentry.Unity/SentryUnity.cs

```diff
@@ -29,24 +29,9 @@ public static void Init(SentryUnityOptions unitySentryOptions)
         {
             unitySentryOptions.TryAttachLogger();
 
-            // Uses the game `version` as Release unless the user defined one via the Options
-            if (unitySentryOptions.Release == null)
-            {
-                unitySentryOptions.Release = Application.productName is string productName
-                    && !string.IsNullOrWhiteSpace(productName)
-                        ? $"{productName}@{Application.version}"
-                        : $"{Application.version}";
-
-                unitySentryOptions.DiagnosticLogger?.Log(SentryLevel.Debug,
-                    "Setting Sentry Release to Unity App.Version: {0}",
-                    null, unitySentryOptions.Release);
-            }
-
-            unitySentryOptions.Environment = unitySentryOptions.Environment is { } environment
-                ? environment
-                : Application.isEditor // TODO: Should we move it out and use via IApplication something?
-                    ? "editor"
-                    : "production";
+            SentryDefaultOptionSetter.SetRelease(unitySentryOptions);
```

Comment:
What about a single method that sets all defaults?

`SentryDefaultOptionSetter.SetDefaults(options)`